cypress/e2e/partial_import_test.spec.ts

```diff
@@ -14,16 +14,18 @@ describe("Partial import test", () => {
   const realmSettings = new RealmSettings();
 
   beforeEach(() => {
-    keycloakBefore();
-    loginPage.logIn();
-    sidebarPage.goToRealm(TEST_REALM);
     sidebarPage.goToRealmSettings();
     realmSettings.clickActionMenu();
   });
 
-  before(async () => {
-    await adminClient.createRealm(TEST_REALM);
-    await adminClient.createRealm(TEST_REALM_2);
+  before(() => {
+    cy.wrap(null).then(async () => {
+      await adminClient.createRealm(TEST_REALM);
+      await adminClient.createRealm(TEST_REALM_2);
+    });
```

Comment:
This could be done in parralel.
```suggestion
    cy.wrap(null).then(() => Promise.all([
      adminClient.createRealm(TEST_REALM),
      adminClient.createRealm(TEST_REALM_2)
    ]);
```